ci/test-checks.sh

```diff
@@ -86,7 +86,13 @@ nightly_clippy_allows=(
    "${nightly_clippy_allows[@]}"
 
 _ scripts/cargo-for-all-lock-files.sh -- nightly sort --workspace --check
-_ scripts/cargo-for-all-lock-files.sh -- nightly fmt --all -- --check
+_ scripts/cargo-for-all-lock-files.sh -- nightly fmt --all -- --check | tee nightly_fmt_check_result
```

Comment:
what's the tee for?

Reply:
we set -e in the begin. I would like to hide the error to avoid it exit immediately. Maybe there is another better way 🤔

Reply:
@t-nelson what do you think? If you don't oppose, I'm going to clean and merge this PR. or maybe we don't need this feature anymore? 🙊